### DISCUSSION TITLE
Role Based Navigation Layout

### DISCUSSION THREAD

**Discussion by Christianoooooo** [Budibase/budibase#6281] [Ideas]
I would like to have a navigation layout that works by role, meaning the admin sees "Home, Ticket System, Admin" in the navigation.

The Public User, but only "Home, Ticket-System".

Do you understand what I mean?

**Comment by Christianoooooo** [comment]
This should work both with the standard roles and with the custom roles

**Comment by melohagan** [comment]
Hi @EinfachChrisii 

Yes this is possible, I've updated the docs slightly so hopefully it'll be a little clearer: https://docs.budibase.com/docs/screens#access-level

**Comment by Christianoooooo** [comment]
Hey @melohagan 
Can I also use it to hide certain entries in the navigation? That Public User e.g. only see "Beweiserfassung" and "Beweisführung" only Admin User see in the navigation.

![raw (1)](https://user-images.githubusercontent.com/107344603/173328571-5832fb3e-a943-4ba3-8796-3b9df8ebdd3d.png)

**Reply by melohagan** [reply]
Unfortunately not for navigation links at the moment. 

You can use Conditional UI and the role bindings to hide/show links in your screen based on role: https://docs.budibase.com/docs/conditions

In future, the UI will include a minimum role setting beside each link which would be a handy shortcut to using conditional ui.

**Reply by NicolasFabre81** [reply]
Hello @EinfachChrisii 
you can manage it by making an "admin" layout with the links to the admin-only screens.
Then you create an admin screen with the admin layout in your app, and you set this screen access to power user role for example.
Other users will get the default page.
